### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by solarbeam**

### CODE REVIEW

File: TOPBanga/Detection/PositionLogger.cs [hunk 1]
```diff
@@ -1,57 +1,73 @@
...
-namespace TOPBanga.Detection
+namespace TOPBanga
 {
     class PositionLogger
     {
         /**
          * Defines the object's position on the x axis
          **/
-        public int posX { get; private set; }
+        public float posX { get; private set; }
```
Comment (mrzilvis) [inline TOPBanga/Detection/PositionLogger.cs]: Use first upperletter

File: TOPBanga/Detection/PositionLogger.cs [hunk 2]
```diff
@@ -1,57 +1,73 @@
...
         /**
          * Defines the object's position on the y axis
          **/
-        public int posY { get; private set; }
+        public float posY { get; private set; }
```
Comment (mrzilvis) [inline TOPBanga/Detection/PositionLogger.cs]: Use first upperletter

File: TOPBanga/Detection/ColorDetector.cs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+namespace TOPBanga
+{
+    class ColorDetector : IDetector
+    {
+        public Image<Bgr, byte> image { get; set; }
+
+        public Hsv ballHsv { get; set; }
+
+        public int threshold { get; set; }
```
[the remark this answers is not in the document]
Comment (solarbeam) [reply TOPBanga/Detection/ColorDetector.cs]: http://www.dofactory.com/reference/csharp-coding-standards

According to this standard, this is fine.

File: TOPBanga/Detection/PositionLogger.cs (hunk 1 above)
Comment (solarbeam) [reply TOPBanga/Detection/PositionLogger.cs]: This code is deprecated and will probably deleted in the next merge.

File: TOPBanga/Detection/PositionLogger.cs (hunk 2 above)
Comment (solarbeam) [reply TOPBanga/Detection/PositionLogger.cs]: This code is deprecated and will probably deleted in the next merge.

**Pull request merged**